_posts/2016-06-17-cynthia-intro.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Cynthia: introduction"
```

Comment:
I think this would look best if this was `Cynthia Vint: Introduction`, then you can remove your name below.